test/units/cli/test_galaxy.py

```diff
@@ -585,8 +585,6 @@ def test_invalid_skeleton_path():
     "hypen-ns.collection",
     "ns.hyphen-collection",
     "ns.collection.weird",
-    "Ånsible.collection",
```

Comment:
For such things, it's always better to use an explicit Unicode string literals (i.e. `u"Ånsible.collection"`)